### DISCUSSION TITLE
Sveltekit deployment not working

### DISCUSSION THREAD

**Discussion by alexbjorlig** [vercel/community#3329] [Help] [answered]
### Summary

When I deploy this Sveltekit boiler project on Vercel, the build script itself works.
But when I visit the app, I just get:

<img width="583" alt="CleanShot 2023-07-21 at 13 36 49@2x" src="https://github.com/vercel/community/assets/9967422/a348f013-80b3-4a86-949d-5850d3e8a9f6">

I hope you can help solve this. Please note that the Sveltekit boiler project has no changes, it's just nested under `apps/web-app` folder. Thanks 🙏

### Example

https://github.com/21RISK/sveltekit-boilerplate

### Steps to Reproduce

Deploy the GitHub [repo](https://github.com/21RISK/sveltekit-boilerplate) to Vercel, with the following Build & Development Settings:

<img width="926" alt="CleanShot 2023-07-21 at 13 40 24@2x" src="https://github.com/vercel/community/assets/9967422/4e63834c-b329-4920-a856-6c7e3ee4eb74">

**Please note**
The reason why I'm not using the "Root Directory" option, is because in my original problem, that solution does not work. And if I understand correctly, it should be possible to specify the output directly.

**Comment by amyegan** [comment, marked as answer]
Hey @alexbjorlig. It looks like you may have a misconfigured monorepo. I recommend reading through the [Monorepo](https://vercel.com/docs/concepts/monorepos) documentation. 

If you're using a modified version of the [Turborepo & SvelteKit Starter](https://vercel.com/templates/svelte/turborepo-sveltekit-starter) template, then the root directory should be `apps/web-app` and "Include source files outside of the Root Directory in the Build Step" should be checked. `SvelteKit` should be selected as the Framework Preset with `cd ../.. && pnpm turbo build --filter=web...` (or whatever build command override your app needs). 

<img width="929" alt="Default build and development settings for turbo repo svelte kit starter template" src="https://github.com/vercel/community/assets/8247105/c8aac562-bf21-4111-a52b-f81415fbc4e0">

**Reply by alexbjorlig** [reply]
Ok, in the repo project, your suggested setting actually does work (and I will try to use this setting in my real project).

But help me out understanding the "Output directory" option then. Why does my approach not work? Is it not so simple that I can point to a specific folder with the build artificats from the vercel adapter?
